packages/astro/test/config-vite.test.js

```diff
@@ -21,3 +23,14 @@ describe('Vite Config', async () => {
 		assert.match($('link').attr('href'), /\/assets\/testing-[a-z\d]+\.css/);
 	});
 });
+
+describe("getViteConfig", () => {
+	it("Does not change the default config.", async () => {
+		const command = "serve";
+		const mode = "test";
+		const configFn = getViteConfig({});
+		const config = await configFn({ command, mode });
+		const resolvedConfig = await resolveConfig(config, command, mode);
+		assert.deepStrictEqual(resolvedConfig.resolve.conditions, ["astro"]);
```

Comment:
Before the fix, the test fails as follows. (The contents of the array are duplicated)

![image](https://github.com/user-attachments/assets/a8b6d4b9-f588-45e7-98ed-2180a3f3bee3)